SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -1,6 +1,6 @@
 // Declare app level module which depends on filters, and services
 angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date',
-    'routeStyles', 'angular-jwt', 'ngWebCrypto', 'angular-uuid'])
+    'routeStyles', 'angular-jwt', 'ngWebCrypto', 'angular-uuid', 'datatables'])
```

Comment:
This should solve that error we were seeing before when using datatables within the app 👍

Reply:
correct 👍